category.proto

```diff
@@ -495,4 +495,10 @@ enum Category {
   KIDTRAVEL_4  = 10487;
   UKTRAVEL_4   = 10488;
   TRAVEL_4     = 10489;
+
```

Comment:
We don't yet know that these are the final names of the categories, or that their IDs match legacy.